app/helpers/alchemy/admin/essences_helper.rb

```diff
@@ -80,6 +80,25 @@ def essence_picture_thumbnail(content, options)
         )
       end
 
+      # Show image cropping link for given content and options?
+      def allow_image_cropping?(content, options = {})
+        content.settings_value(:crop, options) &&
+          content.ingredient &&
+            content.ingredient.can_be_cropped_to(
```

Comment:
Style/MultilineOperationIndentation: Use 2 (not 4) spaces for indenting an expression spanning multiple lines.